packages/genesys-spark-components/src/components/beta/gux-blank-state-beta/gux-illustration/svg-utils.tsx

```diff
@@ -0,0 +1,1039 @@
+import { JSX, h } from '@stencil/core';
```

Comment:
I think the performance hit of doing it this way might not be worth it.
Did you try to do it the same way as gux-icon?